pkg/components/components.go

```diff
@@ -529,6 +529,7 @@ func GetCSVBase(name, hcCRNamespace, displayName, description, image, replaces s
 				"description":    description,
 				"repository":     "https://github.com/kubevirt/hyperconverged-cluster-operator",
 				"support":        "false",
+				"operatorframework.io/suggested-namespace": hcCRNamespace,
```

Comment:
This is already addressed in #461 : can you please rebase this on master and squash in a single commit ?